utils/docker/Dockerfile.centos.7

```diff
@@ -81,7 +82,8 @@ RUN pip3 install avocado-framework; \
     pip3 install avocado-framework-plugin-result-html; \
     pip3 install avocado-framework-plugin-varianter-yaml-to-mux
 
-RUN pip install codespell
+RUN pip3 install codespell
+RUN pip3 install pygithub
```

Comment:
Why not use:
```
python36-pygithub.noarch : Python library to work with the Github API
```
Instead?

Reply:
The code_review PR monkey-patches pygithub so is fairly tightly tied to recent versions of the package.

Reply:
Which PR is that?  I would argue that if we are monkey-patching something, that makes it all that more important to be installing a consistent version, from an RPM, if we have one available.  `pip install` will always install the latest version which could change at any time, unexpectedly, breaking the monkey-patching.

Reply:
https://github.com/daos-stack/code_review/pull/23